### PR TITLE
added GIFs to animate dynamic highline

### LINKED ISSUE #128: Add GIF to animated dynamic highline
From #127 

> I think ut would be really cool, if we could include a GIF of the animated bridge for the correct solution, just like in the first exercise. Preferably, this GIF is in the same style, so that the differences in the static and dynamic models are directly apparent.

### PR DESCRIPTION
**Pull request by PhiSpel**
and changed some parameters to make it comparable

closes #128

### CODE REVIEW

File: content/07_num_ode/03_modellierung_einer_highline_dynamisch.ipynb [hunk 1]
```diff
@@ -157,7 +161,24 @@
...
+   },
+   "source": [
+    "```{figure} images/animated_highline.gif\n",
+    "---\n",
+    "name: Highline Animation beim Einschwingen\n",
+    "width: 400px\n",
+    "align: center\n",
+    "---\n",
+    "So sollte die sich einschwingende Highline aussehen\n",
+    "```"
```
Comment (sandro-elsweijer) [inline content/07_num_ode/03_modellierung_einer_highline_dynamisch.ipynb:176]: This format introduces labels, which we don't have on the other pages. Also, the label starts with Fig.7 and I don't know why.
On the other pages we use this format:
````
```{image} path/to/image.png
:alt: Name of image
:width: 1200px
:align: center
```
````
I will add this to our contributing guide

File: content/07_num_ode/03_modellierung_einer_highline_dynamisch.ipynb [hunk 2]
```diff
@@ -308,18 +331,50 @@
...
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "```{figure} images/animated_highline_with_wind.gif\n",
+    "---\n",
+    "name: Highline Animation mit Wind\n",
+    "width: 800px\n",
+    "align: center\n",
+    "---\n",
```
Comment (sandro-elsweijer) [inline content/07_num_ode/03_modellierung_einer_highline_dynamisch.ipynb:355]: see above

File: content/07_num_ode/03_modellierung_einer_highline_dynamisch.ipynb [hunk 3]
```diff
@@ -308,18 +331,50 @@
...
+   "cell_type": "markdown",
+   "id": "42f11343",
+   "metadata": {},
+   "source": [
+    "```{figure} images/animated_highline_with_person.gif\n",
+    "---\n",
+    "name: Highline Animation mit Highliner\n",
+    "width: 800px\n",
+    "align: center\n",
+    "---\n",
```
Comment (sandro-elsweijer) [inline content/07_num_ode/03_modellierung_einer_highline_dynamisch.ipynb:370]: see above

File: content/07_num_ode/03_modellierung_einer_highline_dynamisch.ipynb [hunk 4]
```diff
@@ -332,6 +387,19 @@
...
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ffe9a295",
+   "metadata": {},
+   "source": [
+    "## Aufgabe 4\n",
+    "\n",
+    "Reale Slacklines haben eine Dehung von etwa 15 % bei 15 kN (siehe [hier](https://www.slacklineverband.com/know-how/slackline-vergleich-dehnung/) und [hier](https://www.slacklineverband.com/know-how/slackline-vergleich-gewichtbruchlast/)) und ein Gewicht von 60 g/m.\n",
```
Comment (sandro-elsweijer) [inline content/07_num_ode/03_modellierung_einer_highline_dynamisch.ipynb:398]: I didn't know tests like these exist, looks great!

**Review by sandro-elsweijer — CHANGES_REQUESTED** [review]

File: content/07_num_ode/03_modellierung_einer_highline_dynamisch.ipynb [hunk 5]
```diff
@@ -13,40 +13,44 @@
...
-   "id": "7495658f",
+   "id": "a631b19c",
    "metadata": {},
    "source": [
     "## Aufgabe 1\n",
     "\n",
     "Lösen Sie die Differentialgleichung mit der Matlab–Routine `ode45` auf einem Zeitinterval von 90 Sekunden. Als externte Kraft $\\mathbf{F}_{ext}$ soll zunächst nur die Schwerkraft auf die Brückenknoten wirken. Verwenden Sie $\\mathbf{z}(0)=\\mathbf{\\dot{z}}(0)=0$ als Anfangsbedingungen. Zunächst nehmen wir $\\mathbf{F}_{ext} = \\text{const.} = \\mathbf{F}_g$ an.\n",
     "\n",
-    "### Tipps:\n",
+    "```{admonition} Tipps:\n",
```
Comment (sandro-elsweijer) [inline content/07_num_ode/03_modellierung_einer_highline_dynamisch.ipynb:35]: For consistency, we should use the singular and leave out the `:`

**Review by sandro-elsweijer — CHANGES_REQUESTED** [review]

File: content/07_num_ode/03_modellierung_einer_highline_dynamisch.ipynb (hunk 1 above)
Comment (PhiSpel) [reply content/07_num_ode/03_modellierung_einer_highline_dynamisch.ipynb]: With images, you cannot add captions. I can also just add some text above. See c5658fb

Which looks better?

File: content/07_num_ode/03_modellierung_einer_highline_dynamisch.ipynb (hunk 5 above)
Comment (PhiSpel) [reply content/07_num_ode/03_modellierung_einer_highline_dynamisch.ipynb]: Good point. See df6cebf

File: content/07_num_ode/03_modellierung_einer_highline_dynamisch.ipynb (hunk 1 above)
Comment (sandro-elsweijer) [reply content/07_num_ode/03_modellierung_einer_highline_dynamisch.ipynb]: I think this looks good

**Review by sandro-elsweijer — APPROVED** [review]

**Pull request merged**